recipes/pyusb/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyusb" %}
+{% set version = "1.0.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "4e9b72cc4a4205ca64fbf1f3fff39a335512166c151ad103e55c8223ac147362" %}
```

Comment:
Is this `hash_value` variable automatically updated by @regro-cf-autotick-bot, or does it have to be called `sha265` for that, like in the example recipe?

Reply:
I think it may have to be called sha256. There may be other options too, and we are welcome to PRs to improve the bot too!

Reply:
Is there an advantage to being able to change the hash type with a variable instead of set it fixed to sha256? If so, we can indeed improve the bot, else I would suggest to change this recipe according to the example.

Reply:
`hash_value` is fine. https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/migrators.xsh#L118

Reply:
Good to know, thanks!